content-services/content-sources-frontend
# Fixes 4587: ZeroState, Node modules, eslint, fec-config

## Summary
- Fixes a bug in zero-state not appearing due to a mismatch of react-router-dom version changes.
- Bumps a bunch of node_modules.
- The front-end now requires docker or podman (and their corresponding desktop applications) to run locally. 


Additional information:
- While debugging an issue running locally, a [bug](https://github.com/RedHatInsights/frontend-components/pull/2053) was found in the frontend-component library that we use here. 
Until the above bug is merged/frontend-components can be updated, I'll leave this PR in draft state. 


- [x] [Docker bug](https://github.com/RedHatInsights/frontend-components/pull/2053) merged

## Testing steps

- Ensure you have podman/podman desktop installed (or orbstack/docker)
- Delete current node-modules folder (likely not needed but best practice) 
- npm install
- npm start

- Ensure all is working as expected (sanity check)
